1904-12-30.xml

```diff
@@ -0,0 +1,2634 @@
+                    <p> Overture--The Merry Wives of Windsor -- Nicolai] </p>
+                    <p>Valse-Kilnstler Leben ---Strauss. </p>
+                    <p>Selection -San Toy-Jones.</p>
+                    <p> Serenade-Siziletta--Von Blon. </p>
+                    <p>Valse-Enchantée-Berger. </p>
+                    <p>Selection-A Country Girl-Monchtond</p>
+                    <p>(By desire). </p>
+                    <p>Galop Militaire--"Le Diable au Corps" -Gillet.</p>
+                    <p>Regimental March.</p>
+                    <p>Khedivial Hymn</p>
+                    <p>God Save the King R. Watson Ramsey, Bandmaster.</p>
+                </div>
+                <div type="section" feature="passList">
+                    <head>PASSENGER LISTS.</head>
+                    <p>DEPARTURES.</p>
+                    <p>Le paquebot "Cairo" parti hier pour Genes avait à bord :</p>
+                    <p>MM. A. Hassan, E. de Martino, M. et Mme Amelio Thomas, Rev. Trinitad Basurto
+                        et 22 pèlerins mexicains, Frey, Gueral, Moratti, et 32 passagers de 3me
+                        classe.</p>
+                </div>
+                <div type="section" feature="notice">
+                    <head>NOTICE.</head>
+                    <p>The P. &amp; 0. 8. N. Company's Steam Yacht VECTIB," 6,000 tons, will leave
+                        Alexardria at 2 p.m. on Friday, the 30th of Docember, for MARSEILLES, taking
+                        First Saloon Passengers only, at the usual mail</p>
+                    <p>25118-6-4</p>
+                </div>
+                <div type="advert" xml:id="deg-ad-cah01">
+                    <head>CARLTON HOTEL</head>
+                    <p>Bulkeley, Ramleh.</p>
+                    <p>Ten minutes from Alexandria. First-Class in every respect. Very moderate
+                        charges. Bulkeley is the fashionable English quarter. Visitors from Cairo
+                        alight at Sidi Gaber Station.</p>
+                    <p><measure type="indexNo">24480-24-5-905</measure></p>
+                    <p>G. AQUILINA, Proprieter.</p>
+                </div>
+                <div type="advert" xml:id="deg-ad-bcp02">
+                    <head>BECK &amp; CO'S PILSENER BEER</head>
+                    <p>BREMEN.</p>
+                    <p>FEARS NO HONEST COMPETITION FOR QUALITY.</p>
+                    <p>N.B. -- Inferior Brands now being offered to Managers of certain good
+                        circles.</p>
+                    <p>Beware of evilly disposed competitors running down this very Superior Brand
+                        of Beer</p>
+                    <p><measure type="indexNo">xxxxx-xx-x xx</measure></p>
+                </div>
+
+                <cb n="6"/>
+                <div type="section" feature="social">
+                    <head>PERSONAL AND SOCIAL.</head>
+                    <div type="item">
```

Comment:
no need to break personal and social into items--a series of paragraphs will do.